module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -607,7 +625,11 @@ public function getEmailSubject($postParams)
 
         $translated = [];
         foreach ($postParams as $key => $val) {
-            $translated["%%{$key}%%"] = $this->translate($val);
+            $translatedVals = [];
+            foreach (!is_array($val) ? [$val] : $val as $v) {
```

Comment:
`foreach ((array)$val as $v) {` should be a slightly more concise way to express this.

...or you might even be able to go fully functional and say something like:

`$translatedVals = array_map([$this, 'translate'], (array)$val);`

Reply:
I agree, edited.